plugins/org.jkiss.dbeaver.cmp.simple.ui/OSGI-INF/l10n/bundle_zh.properties

```diff
@@ -1,5 +1,7 @@
+Bundle-Vendor = JKISS
+Bundle-Name = DBeaver\u6a21\u5f0f\u6bd4\u8f83UI(\u7b80\u5355)
 
-command.org.jkiss.dbeaver.core.compare.simple.description = \u6570\u636E\u5E93\u5BF9\u8C61\u7ED3\u6784\u7684\u7B80\u5355\u6BD4\u8F83
-command.org.jkiss.dbeaver.core.compare.simple.name        = \u7B80\u5355\u7ED3\u6784\u6BD4\u8F83
+command.org.jkiss.dbeaver.core.compare.simple.name        = \u7b80\u5355\u7ed3\u6784\u6bd4\u8f83
+command.org.jkiss.dbeaver.core.compare.simple.description = \u6570\u636e\u5e93\u5bf9\u8c61\u7ed3\u6784\u7684\u7b80\u5355\u6bd4\u8f83
 
-menu.compare.label = \u6BD4\u8F83/\u8FC1\u79FB(&P)
+menu.compare.label = \u6bd4\u8f83/\u8fc1\u79fb(&P)
```

Comment:
A bunch of changes is just lowercasing ascii codes. It was made by the tool you used, Could you set it up to stop doing that? It's really impossible to see the changes you've made and review them, because everything is marked as changed. Please let me know if you are unaware of how to do it, I'll try to help.